lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
@lau this will obviously conflict with the calendar library. Luckily, the Calendar module in `Calendar` is mostly a facade module and defines only one macro. In any case, I would like to discuss this conflict explicitly. We have a couple options:

1. We can rename the `Calendar` module in Elixir to something else.

2. We can rename the `Calendar` module in `calendar` to something else, like `Calendar.Types`.

3. We can add the `__using__` function currently defined in your Calendar module to this module with a deprecation warning. This means that, when users migrate to Elixir 1.3, if they `use Calendar`, they will get a proper deprecation warning. We would remove the deprecation by Elixir 1.4.